draft-ietf-rats-endorsements.md

```diff
@@ -219,6 +219,11 @@ multiple parsers for matching policies.
 
 # Endorsing Verification Keys
 
+{{Section 8.4 of RFC9334}} discusses how a Verifier stores one or more trust anchors
+in its trust anchor store.  The Verifier's trust in an Endorser is expressed via
+storing a trust anchor for the Endorser.  The binding from an Endorsement to
+a given Target Environment is done as follows.
```

Comment:
This new paragraph doesn't flow well in the context of the section.

Reading Carl's comment:

> "[...] I think it may benefit from a bit more discussion on how trust is established in endorsements and how the binding from endorsements to target environment is established. There is no mention of either in the security considerations section. RFC 9334 defines endorsements as being a “secure statement”, so piggybacking off that definition to reference some (abstract) state that must be present in the verifier to verify the endorsement may suffice."

he wanted us to add the missing prose to the security considerations section instead of Section 4.

Reply:
Ok moved this into PR #43

Reply:
thank you!